src/documentation/pages/components/transfer-list/_properties.vue

```diff
@@ -0,0 +1,84 @@
+<template lang="pug">
+div
```

Comment:
No need for this div! Since vue3, templates can contain more than one root element

Reply:
Thanks, fixed.